amazon-link-engine/trunk/amazon-link-engine.php

```diff
@@ -276,9 +293,16 @@ function genius_ale() {
   <script src="//cdn.georiot.com/snippet.min.js" defer></script>
   <script type="text/javascript">
     jQuery(document).ready(function( $ ) {
-      if (typeof Georiot !== 'undefined') {
-        Georiot.amazon.convertToGeoRiotLinks(<?php echo $gr_use_tsid ?>, <?php print($preserve_tracking)?><?php print($gr_use_domain) ?>);
-      };
+	  var ale_on_click_checkbox_is_checked="<?php echo ale_get_on_click_checkbox_state();?>";
+
+	  if (typeof Georiot !== 'undefined') {
```

Comment:
You don't need this shield if that has not loaded by the time the document has loaded the shield will just make us fail silently